src/context/auth-context/auth-context.js

```diff
@@ -20,12 +20,14 @@ function AuthProvider({ children }) {
           const response = await axios.post("/api/auth/verify", {
             encodedToken: smashCartToken,
           });
-          console.log(response);
           if (response.status === 200) {
             setUser(response.data.user);
           }
         } catch (error) {
-          console.error(error.response);
+          dispatch({
```

Comment:
Suggestion: Instead of dispatching toast everytime you can create a utility function like this
![image](https://user-images.githubusercontent.com/65125998/161609289-ba9ed6dd-9b5a-4126-a477-4196d3e592ea.png)

Doing this it will be more easier for you, to use your custom made toasts

Reply:
okay its good will look into it